test/test-generate-pras.jl

```diff
@@ -0,0 +1,241 @@
+            end
+        )
+
+    # Make a PRAS System from PSY-4.X System
+    rts_pras_sys = generate_pras_system(rts_da_sys, PSY.Area)
+    @test rts_pras_sys isa PRASInterface.PRAS.SystemModel
+
+    @test test_names_equal(rts_pras_sys.regions.names, area_names)
+    @test test_names_equal(rts_pras_sys.generators.names, generator_names)
+    @test test_names_equal(rts_pras_sys.storages.names, storage_names)
+    @test test_names_equal(rts_pras_sys.generatorstorages.names, generatorstorage_names)
+    @test test_names_equal(rts_pras_sys.lines.names, line_names)
+
+    # Test that timestamps look right
+    # get time series length
+    psy_ts = first(PSY.get_time_series_multiple(rts_da_sys, type=PSY.SingleTimeSeries))
+    @test all(
+        TimeSeries.timestamp(psy_ts.data) .== collect(DateTime.(rts_pras_sys.timestamps)),
+    )
+
+    # Test capacities
+    # 201_HYDRO_4 should have max active power from time series
+    idx = findfirst(x -> x == "201_HYDRO_4", rts_pras_sys.generators.names)
+    hydro_component = PSY.get_component(PSY.HydroDispatch, rts_da_sys, "201_HYDRO_4")
+    max_capacity =
+        floor.(
+            PSY.get_time_series_values(
+                PSY.SingleTimeSeries,
+                hydro_component,
+                "max_active_power",
+            )
+        )
+    @test rts_pras_sys.generators.capacity[idx, 1] == max_capacity[1]
+    @test all(rts_pras_sys.generators.capacity[idx, :] .== max_capacity)
+
+    thermal_component = PSY.get_component(PSY.ThermalStandard, rts_da_sys, "322_CT_6")
+    @test array_all_equal(
+        rts_pras_sys.generators.capacity[
+            findfirst(x -> x == "322_CT_6", rts_pras_sys.generators.names),
+            :,
+        ],
+        Int(floor(PSY.get_max_active_power(thermal_component))),
+    )
+
+    load_values = zeros(Float64, length(rts_pras_sys.regions.names), length(psy_ts))
+    for load in PSY.get_components(PSY.PowerLoad, rts_da_sys)
+        region = PSY.get_area(PSY.get_bus(load))
+        # Fast enough for # areas < 10
+        idx = findfirst(x -> x == PSY.get_name(region), rts_pras_sys.regions.names)
+        max_active_power =
+            PSY.get_time_series_values(PSY.SingleTimeSeries, load, "max_active_power")
+        load_values[idx, :] += max_active_power
+    end
+    @test all(rts_pras_sys.regions.load .== Int.(floor.(load_values)))
+
+    # Test Assess Run
+    sequential_monte_carlo = PRASInterface.PRAS.SequentialMonteCarlo(samples=2, seed=1)
+    shortfalls, = PRASInterface.PRAS.assess(
+        rts_pras_sys,
+        sequential_monte_carlo,
+        PRASInterface.PRAS.Shortfall(),
+    )
+    lole = PRASInterface.PRAS.LOLE(shortfalls)
+    eue = PRASInterface.PRAS.EUE(shortfalls)
+    @test lole isa PRASInterface.PRAS.ReliabilityMetric
+    @test eue isa PRASInterface.PRAS.ReliabilityMetric
+    @test PRASInterface.PRAS.val(lole) >= 0 && PRASInterface.PRAS.val(lole) <= 10
```

Comment:
I'm not sure this test will fail for valid reasons if we modify PSB to include different "static" outage data for RTS-GMLC or if outage time series data leads to different results. Just flagging this for now.

Reply:
Not sure what you mean here. Do you mean you want this test to be more precise then?

Reply:
I think this looks good for now. I am thinking about how this will cange if we change the underlying outage data in the GMLC system.